include/open62541/server.h

```diff
@@ -67,6 +67,11 @@ _UA_BEGIN_DECLS
  *
  * The :ref:`tutorials` provide a good starting point for this. */
 
+#ifdef UA_ENABLE_PUBSUB_MONITORING
+struct UA_PubSubConfiguration;
+typedef struct UA_PubSubConfiguration UA_PubSubConfiguration;
+#endif /* UA_ENABLE_PUBSUB_MONITORING */
+
```

Comment:
The PubSubConfiguration is in the long term not monitoring dependent. I would prefer to always include this type and make just a conditional include around the contained 'UA_PubSubMonitoringInterface monitoringInterface;'.